src/field/gen_fieldops.py

```diff
@@ -164,17 +166,26 @@ def returnType(f1, f2):
     index_var = 'index'
 
     region_name = '"RGN_ALL"'
-    
+
     if args.noOpenMP:
         region_loop = 'BLOCK_REGION_LOOP_SERIAL'
     else:
         region_loop = 'BLOCK_REGION_LOOP'
-        
-    # Declare what fields we currently support:
-    # Field perp is currently missing
+
     field3D = Field('Field3D', ['x', 'y', 'z'], index_var=index_var)
     field2D = Field('Field2D', ['x', 'y'], index_var=index_var)
+    fieldperp = Field('FieldPerp', ['x', 'z'], index_var=index_var)
     boutreal = Field('BoutReal', [], index_var=index_var)
+else:
+    field3D = Field('Field3D', ['x', 'y', 'z'])
+    field2D = Field('Field2D', ['x', 'y'])
+    fieldperp = Field('FieldPerp', ['x', 'z'])
+    boutreal = Field('BoutReal', [])
+
+if __name__ == "__main__":
```

Comment:
This looks like a copy-paste error

Reply:
No - this is intended.
I want to use this Field object also for the flexible field.
This I want to have them set, even in the case of not being `__main__`